Makefile

```diff
@@ -246,43 +246,54 @@ build_demo_app:
 
 
 # nimbus schemas package
-SCHEMAS_VERSION_FILE := schemas/VERSION
-SCHEMAS_VERSION := $(shell cat ${SCHEMAS_VERSION_FILE})
+SCHEMAS_ENV ?=  # This is empty by default
+SCHEMAS_VERSION = \$$(cat VERSION)
+SCHEMAS_RUN = docker run -ti $(SCHEMAS_ENV) -v ./schemas:/schemas -v /schemas/node_modules schemas:dev sh -c
+SCHEMAS_BLACK = poetry run black --check --diff .
```

Comment:
Do we still need `poetry run` for all these commands? I was originally using this so that things would run in the poetry-managed venv, but with docker I think we can just run the commands directly (as long as we install them in the container).